src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -607,8 +607,10 @@ GetItemEntry sGetItemTable[] = {
     GET_ITEM(ITEM_NUT_UPGRADE_30, OBJECT_GI_NUTS, 0x11, 0xA7, 0x80, CHEST_ANIM_SHORT),
     GET_ITEM(ITEM_NUT_UPGRADE_40, OBJECT_GI_NUTS, 0x11, 0xA8, 0x80, CHEST_ANIM_SHORT),
     GET_ITEM(ITEM_BULLET_BAG_50, OBJECT_GI_DEKUPOUCH, 0x72, 0x6C, 0x80, CHEST_ANIM_LONG),
-    { ITEM_NONE },
-    { ITEM_NONE },
+    GET_ITEM(ITEM_NONE, OBJECT_UNSET_0, -1, 0x00, 0x00, CHEST_ANIM_SHORT),
+    GET_ITEM(ITEM_NONE, OBJECT_UNSET_0, -1, 0x00, 0x00, CHEST_ANIM_SHORT),
+    // { ITEM_NONE, 0, 0, 0, 0 },
+    // { ITEM_NONE, 0, 0, 0, 0 },
```

Comment:
`{ ITEM_NONE },` is the best option here imo, but that's assuming we ignore the warning (which we arguably should). The other decent options are to have a `GET_ITEM_NONE` macro for this purpose or to fill the remaining fields with 0, but I really don't think using the normal `GET_ITEM` macro here is good.

Reply:
I don't see the point of having a warning on something that can be resolved simply by padding to agree with the array definition. I don't mind how this is done, but I don't think a warning should be preferred to such an easy fix.

Reply:
a `GET_ITEM_NONE` macro sounds like a good compromise here. agree with roman though that filling in the normal macro does not work. `OBJECT_UNSET_0` and `CHEST_ANIM_SHORT` dont make sense in this context

Reply:
So something like `#define GET_ITEM_NONE { ITEM_NONE, 0, 0, 0, 0 }` after the `CHEST_ANIM_LONG` one at the top?